tests/docker_run_tests.rs

```diff
@@ -228,6 +228,78 @@ fn test_node() {
     assert!(run_image(name, None).contains("Hello from Node"));
 }
 
+#[test]
+fn test_node_nx_next() {
+    let n = create_network();
+    let config = Config {
+        environment_variables: EnvironmentVariables::from([]),
+        network: Some(n.name.to_string()),
```

Comment:
I believe that creating a network here isn't necessary since the container does not interact with any other containers (so it can be `None`).

Reply:
neat, done this now.